guides/common/modules/proc_changing-the-http-proxy-policy-for-a-repository-by-using-cli.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="changing-the-http-proxy-policy-for-a-repository-by-using-cli"]
+= Changing the HTTP proxy policy for a repository by using Hammer CLI
+
+[role="_abstract"]
+For granular control over network traffic, you can set an HTTP proxy policy for each repository.
+
+To set the same HTTP proxy policy for all repositories in a product, see xref:Changing_the_HTTP_Proxy_Policy_for_a_Product_{context}[].
+
+.Procedure
+* On {ProjectServer}, specify the HTTP proxy policy you want to use:
++
+[subs="+quotes"]
+----
+$ hammer repository update \
+--http-proxy-policy _HTTP_Proxy_Policy_ \
+--id _Repository_ID_
+----
++
+Specify one of the following options for `--http-proxy-policy`:
++
+** `none`: Do not use an HTTP proxy, even if a global default proxy is configured.
+** `global_default_http_proxy`: Use the global default proxy setting.
```

Comment:
```suggestion
** `none`: Do not use an HTTP proxy, even if a global default HTTP proxy is configured.
** `global_default_http_proxy`: Use the global default HTTP proxy setting.
```

I highly assume that you did not introduce this wording with your PR. But I always strive to make it clear if we talk about HTTP proxies vs Smart Proxies.